manifests/gateways/istio-egress/templates/deployment.yaml

```
@@ -145,6 +145,8 @@ spec:
 {{- else }}
 {{ toYaml .Values.global.defaultResources | indent 12 }}
 {{- end }}
+          securityContext:
+            runAsUser: 1337
```

Also set the group:
```suggestion
            runAsGroup: 1337
            runAsNonRoot: true
            runAsUser: 1337
```
Same for all the other specs.

---

Got it! Thanks. I'm away for a few days but will get back to this early next week.